From 27a440f42b3c536a2e73004f14b077003bf79207 Mon Sep 17 00:00:00 2001
From: Noam Camus <ncamus@habana.ai>
Date: Sun, 07 Apr 2019 14:59:01 +0300
Subject: [PATCH] [SW-0] spi: update bit operation usage

Use BIT and not BITMASK operation since this is a single bit mask.
Now no more BIT_MASK usage in this project

Change-Id: If08eba284830f5ffe37537716e4d2b2b87976cae
---

diff --git a/common/spi.c b/common/spi.c
index 4deb52a..fd9cddd 100644
--- a/common/spi.c
+++ b/common/spi.c
@@ -51,7 +51,7 @@
  */
 static void spi_ssi_cs0_init(void)
 {
-	unsigned int reg, mask = BIT_MASK(53 % 16);
+	unsigned int reg, mask = BIT(53 % 16);
 
 	reg  = readl(mmPSOC_GPIO3_DATAOUT);
 	reg  &= ~mask;
